test/replication_test.dart

```diff
@@ -0,0 +1,71 @@
+/// Copyright (C) 2022 Intel Corporation
+/// SPDX-License-Identifier: BSD-3-Clause
+///
+/// extend_test.dart
+/// Unit tests for extend and withSet operations
+///
+/// 2023 Jan 18
+/// Author: Akshay Wankhede <akshay.wankhede@intel.com>
+///
+
+import 'package:rohd/rohd.dart';
+import 'package:rohd/src/utilities/simcompare.dart';
+import 'package:test/test.dart';
+
+class ReplicationOpModule extends Module {
+  ReplicationOpModule(Logic a, int multiplier, int newWidth) {
```

Comment:
Why is `newWidth` necessary?  Why can't we just calculate it by multiplying `a.width*multiplier`?